yarn.lock

```diff
@@ -0,0 +1,2783 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
why add yarn.lock?

Reply:
I have used it here as checksum and also, it's not ignored in [the `.gitignore` file](https://github.com/koajs/send/blob/master/.gitignore) and don't exist the `.yarnrc` file too.